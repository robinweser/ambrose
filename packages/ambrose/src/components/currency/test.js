import React from 'react'

import Currency from '.'
import Text from '../text'

import createSnapshot from '../../testing/createSnapshot'

describe('<Currency />', () => {
  it('renders correctly with default props', () => {
    const snapshot = createSnapshot(<Currency>4000.50</Currency>)
    expect(snapshot).toMatchSnapshot()
  })
  it('renders correctly when nested', () => {
    const snapshot = createSnapshot(
      <Text variant="heading">
        <Currency>4000.50</Currency>
      </Text>
    )
    expect(snapshot).toMatchSnapshot()
  })
})
