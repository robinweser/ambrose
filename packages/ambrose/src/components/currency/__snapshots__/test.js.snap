// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Currency /> renders correctly when nested 1`] = `
".cls_0 {
  display: contents;
}
.cls_1 {
  font-size: 20px;
  margin: 0;
  display: block;
  color: black;
}


<div className=test-theme cls_0>
  <div className=cls_1>4&nbsp;000,5 kr</div>
</div>;
"
`;

exports[`<Currency /> renders correctly with default props 1`] = `
".cls_0 {
  display: contents;
}


<div className=test-theme cls_0>4&nbsp;000,5 kr</div>;
"
`;
